datadog_checks_base/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * Fix check cancellation timeout due to `DBMAsyncJob` cancellation being blocked ([#16028](https://github.com/DataDog/integrations-core/pull/16028))
 * Bump the `pyodbc` version to 4.0.39 ([#16021](https://github.com/DataDog/integrations-core/pull/16021))
```

Comment:
As we are directly releasing with pyodbc v5.0.1, can this changelog entry be removed if we haven't released with pyodbc v4.0.39?

Reply:
Good idea! Updated